src/utils/formHelperFunctions.ts

```diff
@@ -785,24 +785,19 @@ export function getInitialStep({
   return initialStepId || sessionCurrentStep || (getOrigin as any)(steps).key;
 }
 
-export function castVal(
-  servarType: string | undefined,
-  val: any,
-  repeated = false
-): any {
+export function castVal(fieldType: string, val: any, repeated = false): any {
```

Comment:
can we make separate functions castServarVal and castHiddenVal instead of putting them in the same one?